sdcm/cluster.py

```diff
@@ -324,8 +324,13 @@ def init(self) -> None:
         if self.node_type == "db" and not self.is_kubernetes() \
                 and self.parent_cluster.params.get("print_kernel_callstack"):
             try:
+                # increase aio-max-nr to a number allowing us to run multiple seastar
+                # applications in parallel, in case we need to run them as command line tools.
+                # otherwise the scylla server would take up all aio slots in system if required
+                # slots is smaller than what the system can provide.
                 self.remoter.sudo(shell_script_cmd("""\
                 echo 'kernel.perf_event_paranoid = 0' >> /etc/sysctl.conf
+                echo 'fs.aio-max-nr=30000000' >> /etc/sysctl.conf
```

Comment:
@tchaikov 

I don't think this is the best place for doing so, here it would be done for all of VMs SCT provision,
and would override any value the scylla RPMs/DEBs would be setting.

I think this should be confined only to offiline install part

Reply:
@fruch thank you for your review! i see. will move this change to a better place.